### PR TITLE
Add tests for hotfix 1.22.1

### PR DESCRIPTION
**Pull request by bdattoma**

### CODE REVIEW

File: tests/Tests/100__deploy/100__installation/102__post_install.robot [hunk 1]
```diff
@@ -281,6 +281,22 @@ Verify RHODS Display Name and Version
     Should Be Equal       ${rhods_version_t[1]}   ${rhods_version}   msg=RHODS vesrion and label is not consistent
     Should Be Equal       ${rhods_displayname}   Red Hat OpenShift Data Science  msg=Dieplay name doesn't match
 
+Verify RHODS Notebooks Network Policies
+    [Documentation]    Verifies that the network policies for RHODS Notebooks are present on the cluster
+    [Tags]    Smoke    TESTME
```
Comment (jgarciao) [inline tests/Tests/100__deploy/100__installation/102__post_install.robot:286]: Can you replace TESTME for Tier1?

File: tests/Tests/500__jupyterhub/networkpolicy-test.robot [hunk 2]
```diff
@@ -0,0 +1,87 @@
+*** Settings ***
+Documentation    Test for the new Notebook network policies (RHODS-6878)
```
Comment (jgarciao) [inline tests/Tests/500__jupyterhub/networkpolicy-test.robot:2]: Could you remove Jira ID?

File: tests/Tests/500__jupyterhub/networkpolicy-test.robot [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+Suite Teardown   Network Policy Suite Teardown
+Force Tags       JupyterHub
+
+
+*** Test Cases ***
+Test Network Policy Effect
+    [Documentation]    Spawns two Notebooks with two different users and verifies
+    ...    That the new network policies prevent user2 from accessing user1's workspace
+    [Tags]  Sanity    Tier1
+    ...    TESTMENB
```
Comment (jgarciao) [inline tests/Tests/500__jupyterhub/networkpolicy-test.robot:19]: Can you remove TESTMENB?

**Review by jgarciao — CHANGES_REQUESTED** [review]

File: tests/Tests/500__jupyterhub/networkpolicy-test.robot (hunk 3 above)
Comment (bdattoma) [reply tests/Tests/500__jupyterhub/networkpolicy-test.robot:19]: yeah waiting for polarion IDs

File: tests/Tests/500__jupyterhub/networkpolicy-test.robot (hunk 2 above)
Comment (bdattoma) [reply tests/Tests/500__jupyterhub/networkpolicy-test.robot]: done

File: tests/Tests/100__deploy/100__installation/102__post_install.robot (hunk 1 above)
Comment (bdattoma) [reply tests/Tests/100__deploy/100__installation/102__post_install.robot:286]: waiting for polarion IDs. I would question also Smoke vs Sanity. WDYT? cc:@lugi0

**Comment by bdattoma** [comment]
Jenkins job on 1.22.1: [1558](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1558/) _RUNNING_

Jenkins job on 1.22.0: coming soon

File: tests/Tests/100__deploy/100__installation/102__post_install.robot (hunk 1 above)
Comment (lugi0) [reply tests/Tests/100__deploy/100__installation/102__post_install.robot:286]: I think this test should run in Smoke.
I haven't been able to access Polarion for the whole afternoon, will provide IDs ASAP

**Comment by tarukumar** [comment]
@bdattoma could you resolve the robocop warning since this is the new script all together.

File: tests/Tests/100__deploy/100__installation/102__post_install.robot (hunk 1 above)
Comment (lugi0) [reply tests/Tests/100__deploy/100__installation/102__post_install.robot:286]: ODS-2045

File: tests/Tests/500__jupyterhub/networkpolicy-test.robot (hunk 3 above)
Comment (lugi0) [reply tests/Tests/500__jupyterhub/networkpolicy-test.robot]: ODS-2046

File: tests/Tests/100__deploy/100__installation/102__post_install.robot (hunk 1 above)
Comment (bdattoma) [reply tests/Tests/100__deploy/100__installation/102__post_install.robot]: done

File: tests/Tests/500__jupyterhub/networkpolicy-test.robot (hunk 3 above)
Comment (bdattoma) [reply tests/Tests/500__jupyterhub/networkpolicy-test.robot]: done

**Comment by bdattoma** [comment]
> @bdattoma could you resolve the robocop warning since this is the new script all together.

I fixed as many as possible. Some of them we cannot fix (e.g., too many keywords)

File: tests/Tests/100__deploy/100__installation/102__post_install.robot [hunk 4]
```diff
@@ -281,6 +281,25 @@ Verify RHODS Display Name and Version
     Should Be Equal       ${rhods_version_t[1]}   ${rhods_version}   msg=RHODS vesrion and label is not consistent
     Should Be Equal       ${rhods_displayname}   Red Hat OpenShift Data Science  msg=Dieplay name doesn't match
 
+Verify RHODS Notebooks Network Policies
+    [Documentation]    Verifies that the network policies for RHODS Notebooks are present on the cluster
+    [Tags]    Smoke    Tier1
```
Comment (lugi0) [inline tests/Tests/100__deploy/100__installation/102__post_install.robot:286]: either smoke or tier1, it shouldn't be both

**Review by lugi0 — CHANGES_REQUESTED** [review]

File: tests/Tests/500__jupyterhub/networkpolicy-test.robot [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+    [Documentation]    Spawns two Notebooks with two different users and verifies
+    ...    That the new network policies prevent user2 from accessing user1's workspace
+    [Tags]  Sanity    Tier1
+    ...     ODS-2046
+    Open Browser And Start Notebook As First User
+    Open Browser And Start Notebook As Second User With Env Vars
+    Clone Git Repository And Run    https://github.com/redhat-rhods-qe/ods-ci-notebooks-main    ods-ci-notebooks-main/notebooks/500__jupyterhub/api/notebook_access.ipynb
+    Sleep  1
+    JupyterLab Code Cell Error Output Should Not Be Visible
+    Sleep    5s
```
Comment (lugi0) [inline tests/Tests/500__jupyterhub/networkpolicy-test.robot:25]: these can be removed I think

Comment (bdattoma) [reply tests/Tests/500__jupyterhub/networkpolicy-test.robot]: done

File: tests/Tests/100__deploy/100__installation/102__post_install.robot (hunk 4 above)
Comment (bdattoma) [reply tests/Tests/100__deploy/100__installation/102__post_install.robot]: done

**Review by lugi0 — APPROVED** [review]

**Review by jgarciao — APPROVED** [review]

**Pull request merged**